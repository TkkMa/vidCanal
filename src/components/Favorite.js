import React from 'react';
import FavoriteListFilters from './FavoriteListFilters';
import FavoriteList from './FavoriteList';

const Favorite = ()=>{
    return(
        <div>
            <FavoriteListFilters />
            <FavoriteList />
        </div>
    )
}

export default Favorite