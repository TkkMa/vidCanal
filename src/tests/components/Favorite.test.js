import React from 'react';
import {shallow} from 'enzyme';
import Favorite from '../../components/Favorite';

test('should render Favorite page correctly', ()=>{
    const wrapper = shallow(<Favorite />);
    expect(wrapper).toMatchSnapshot();
})