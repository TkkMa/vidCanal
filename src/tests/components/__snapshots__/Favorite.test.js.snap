// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Favorite page correctly 1`] = `
<div>
  <Connect(FavoriteListFilters) />
  <Connect(FavoriteList) />
</div>
`;
